tests/test_edge_cases.py

```diff
@@ -141,7 +141,7 @@ def test_indexes():
     )
     clinical_df = pd.DataFrame({"condition": [0, 1]}, index=["sample01", "sample02"])
 
-    with pytest.raises(ValueError):
+    with pytest.raises((KeyError, AssertionError)):
```

Comment:
Shouldn't raise an `AssertionError` anymore.

```suggestion
    with pytest.raises(KeyError):
```